Device specific information for DCM Floodlight (#523)
rudderlabs/rudderstack-docs: destinations/advertising/dcm-floodlight.mdx

| `properties.quantity` | `qty` | Optional | Sales |
| `properties.revenue` | `cost` | Optional | Sales |

<div class="infoBlock">
    

> You must send the device-specific information for DCM Floodlight, such as <code class="inline-code">IDFA</code> or <code class="inline-code">advertisingId</code>, which are mapped to the <code class="inline-code">context.device.advertisingId</code>. You can do this by using the RudderStack's mobile SDK as it collects this information automatically. However, in case you are using a RudderStack server-side SDK, the <code class="inline-code">advertisingId</code> needs to be included manually.

Reviewer: ```suggestion
You must send the device-specific information for DCM Floodlight like <code class="inline-code">IDFA</code> or <code class="inline-code">advertisingId</code>, which are mapped to the <code class="inline-code">context.device.advertisingId</code>. You can do this using RudderStack's mobile SDKs as they collect this information automatically. In case you are using a RudderStack server-side SDK, the <code class="inline-code">advertisingId</code> needs to be included manually.
```

Reviewer: Would suggest including this tip as a separate warning note:

In case you are using a RudderStack server-side SDK, the <code class="inline-code">advertisingId</code> needs to be included manually.

| `properties.quantity` | `qty` | Optional | Sales |
| `properties.revenue` | `cost` | Optional | Sales |

<div class="infoBlock">
    

> You must send the device-specific information for DCM Floodlight, such as <code class="inline-code">IDFA</code> or <code class="inline-code">advertisingId</code> . This can be done using the RudderStack's mobile SDK as it collects this information automatically. However, in case you are using a RudderStack server-side SDK, the <code class="inline-code">advertisingId</code> needs to be included manually.

Reviewer 2: This is should be mapped to `context.device.advertisingId` can we mention that as well

Author: done
